recipe/meta.yaml

```diff
@@ -22,14 +22,15 @@ requirements:
     - {{ compiler("c") }}
     - {{ stdlib("c") }}
     - {{ compiler("cxx") }}
-    - {{ cdt("systemd-devel") }}
     - cmake
     - ninja
-    - libudev
-    - libsystemd
     - pandoc
+    - python
+    - docutils
   host:
     - libnl
+    - libsystemd
+    - libudev
```

Comment:
These were in `requirements/build` before. However the build links to them. So these should be in `requirements/host`. Hence this move

With these changes we now get

```
-- Found UDev: $PREFIX/lib/libudev.so
...
-- Found Systemd: $PREFIX/lib/libsystemd.so
```